stateful-datatable-demo/src/com/axonivy/market/statefuldatatable/entity/BusinessObject.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * 
+ */
+package com.axonivy.market.statefuldatatable.entity;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import com.axonivy.market.statefuldatatable.enums.Availability;
+import com.axonivy.market.statefuldatatable.enums.BusinessObjectStatus;
+import com.axonivy.market.statefuldatatable.enums.Quality;
+
+/**
+ * @author david.merunko
+ *
+ */
+@Entity
+public class BusinessObject extends AbstractEntity {
+	private static final long serialVersionUID = 1L;
+
+	@Id
+	@Column(length = 32, nullable = false) 
+	@GeneratedValue(generator="system-uuid")
+    @GenericGenerator(name="system-uuid", strategy = "uuid")
+	private String id;
+	//Adding Column STEP 1

```

Comment:
This will guide the developer a long with the [how-to-add-a-colum](https://github.com/majeed-hm/stateful-datatable-demo/blob/master/stateful-datatable-product/README.md#how-to-add-a-colum)